azurelinuxagent/ga/collect_telemetry_events.py

```diff
@@ -323,15 +325,20 @@ def _parse_event_and_ensure_it_is_valid(self, extension_event):
         :param extension_event: The json event from file
         :return: Verified Json event that qualifies the contract.
         """
-
-        clean_string = lambda x: x.strip() if x is not None else x
+        def _clean_value(k, v):
+            if v is not None:
```

Comment:
if v is None, _clean_value will return None and the conversion below will raise an exception

Reply:
Sorry, where? I kept it same as we have today so that downstream code will work like before.

Reply:
never mind, you are converting the key, not the value